content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -458,6 +458,16 @@ of individual policies are not defined here.
 - {{< example file="policy/baseline-psp.yaml" >}}Baseline{{< /example >}}
 - {{< example file="policy/restricted-psp.yaml" >}}Restricted{{< /example >}}
 
+## Alternatives
```

Comment:
```suggestion
### Alternatives
```
Based on the description of this PR, do you suggest to have `Alternatives` under `Policy Instantiation`?
Currently, `Alternatives` would be its own section